BaSP-a2022/ayom-a-trackgenix-server
# BaSP-TG-116: Implement firebase server

- Install Firebase library
- Set .env
- Configure, initialize and export Firebase app
- Create middleware to check the Firebase user token and role
- Update controllers for all user types: create, update and delete requests
- Add role permissions to every request

QA: 
- Checkout master
- Update master  `git fetch && git pull origin master`
- Checkout to this branch ` git checkout feature/TG-116/implement-firebase-server` 
- Install the dependencies `npm i`
- Run in console `npm start`
- See the description in the PR of the ticket TG-115 to make the frontEnd side of the test
